.github/workflows/lint.yml

```diff
@@ -0,0 +1,46 @@
+name: lint
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: check out latest commit
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref || github.ref }}
+          token: ${{ secrets.GH_DEPLOY_TOKEN }}
+
+      - name: cache node_modules
+        uses: actions/cache@v2
+        id: cache-node-modules
+        with:
+          path: node_modules/
+          key: reactui-modules-${{ hashFiles('yarn.lock') }}
+
+      - name: get node version
+        id: node-version
+        run: echo "::set-output name=value::$( cat .node-version )"
+
+      - name: set up node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.node-version.outputs.value }}
+          registry-url: https://npm.pkg.github.com
+          scope: '@rentpath'
+          token: ${{ secrets.GH_GPR_TOKEN }}
+
+      - name: install dependencies
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GH_GPR_TOKEN }}
+        run: |
+          yarn config set @rentpath:registry https://npm.pkg.github.com
+          yarn --frozen-lockfile --production=false
+
+      - name: lint
+        run: yarn run lint:js
```

Comment:
Maybe I'm missing it, but this doesn't replace `yarn lint`. This skips part of the linting process.

https://github.com/rentpath/react-ui/blob/master/package.json#L16

Reply:
That's a fair cop. I don't know why I didn't call the same `yarn` script that's called in `.travis.yml`.

But now that you mention it, I'd rather have style linting in a separate job so it can run in parallel with the code lint.